src/ts/helper/AdHelper.ts

```diff
@@ -1,8 +1,47 @@
 import { Ad, AdBreak, AdBreakEvent, AdData, AdEvent, ErrorEvent, LinearAd, PlayerAPI, VastAdData } from 'bitmovin-player';
 import * as Conviva from '@convivainc/conviva-js-coresdk';
+import { INTEGRATION_VERSION_CONTENT_METADATA_CUSTOM_TAG } from '../ConvivaAnalyticsTracker';
+
+export interface ServerSideAdInfo {
```

Comment:
NIT: this is called `SsaiAdInfo` on iOS, maybe we can align here.

Reply:
Thank you for looking into PR! I've renamed to `SsaiAdInfo` - https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/112/commits/8a5eb0e499765f27d78a3051b0dd42ca7a6a4e00.